curriculum/challenges/english/25-front-end-development/lab-confidential-email-page/66bba6fff611169359d9d36a.md

```diff
@@ -0,0 +1,249 @@
+---
+id: 66bba6fff611169359d9d36a
+title: Build a Confidential Email Page
+challengeType: 14
+dashedName: build-a-confidential-email-page
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have a `main` element with an `id` of `email`.
+1. Your `#email` element should have `padding` of `50px`, a `top-margin` of `50px`, a `width` of `500px`, and a `border` that's `2px`.
+1. The total width of your `#email` element should be `500px`.
+1. You should have two `div` elements, one with an id of `confidential` and the other with an id of `top-secret`, within your `#email` element.
+1. Your `#confidential` and `#top-secret` elements should have a `display` of `inline-block`.
+1. Your `#confidential` and `#top-secret` elements should have a `padding`, a `left-margin`, and a `border`.
```

Comment:
```suggestion
1. Your `#confidential` and `#top-secret` elements should have a `padding`, a left margin, and a `border`.
```
I'd use either `margin-left` between backticks or "margin left" as simple text. The same holds for other occurrences.